### DISCUSSION TITLE
Can we reduce the green border size?

### DISCUSSION THREAD

**Discussion by SuperSwap** [tom-englert/RegionToShare#38] [Q&A]
Can we reduce the green border size?

I checked settings but didn't find it.

**Comment by tom-englert** [comment]
What would that help with?

**Reply by phillypro** [reply]
for me....i want the green border gone lol
im seeing if i can use this to rebroadcast a projector and i didnt realize the big green border is actually part of the render when you share your screen....figured this app would try to capture "within" the green....as opposed to the green itself 

EDIT:  Nevermind i found that regiontoshare creates 2 share options....one with the border and one without....i was using the wrong one

**Reply by tom-englert** [reply]
That's why I always recommend to use is like documented, and not to first start active mode and then select the window to share.

**Comment by tom-englert** [comment]
@SuperSwap looks like you are sharing the wrong window, too.

**Reply by phillypro** [reply]
yea he most likely is
i was wondering.....and this is the only thing i can think of
but how possible is it to launch this program already rendering....without the need to click the play button?
it remembers its previous size and position which i assume is to speed things up....but what about also remembering its state?

i was able to struggle together a method of launching this app via command line even tho its a microsoft store app which made it reaaaaaaaallly difficult lol

since i cant add command line arguements.....that one setting would be amazing lol

**Reply by tom-englert** [reply]
That's the same discussion as #31, starting in active mode makes it much harder to use or even use it wrong (s.a.) - just to save one mouse click!

**Reply by phillypro** [reply]
last question....is it possible to use this as a virtual camera?
it would be incredible if that functionality was added....

**Reply by tom-englert** [reply]
What's a "virtual camera"?

**Reply by phillypro** [reply]
to get an idea of what a virtual camera is .....download the free trial of manycams software
basically it does what your app does.....but instead of creating a region of screenspace as its own screen output
it makes it a webcam device output :)
